src/visualization/types.jl

```diff
@@ -348,58 +348,82 @@ end
 
 # specializes the PlotData2D constructor to return an PlotData2DTriangulated for any type of mesh.
 function PlotData2DTriangulated(u, mesh, equations, dg::DGSEM, cache;
-                                solution_variables=nothing, nvisnodes=2*polydeg(dg))
+                                solution_variables=nothing, nvisnodes=nothing,
+                                elevations=nothing, slice=:yz, point=(0,0,0))
```

Comment:
Do we have some documentation of this new functionality (including the keyword argument `elevations` and the specific behavior of `nvisnodes`? Could you also please add some entry in NEWS.md?